engine/src/main/java/org/terasology/rendering/world/WorldRendererLwjgl.java

```diff
@@ -193,31 +162,26 @@ public boolean pregenerateChunks() {
     }
 
     @Override
-    public void update(float delta) {
+    public void update(float deltaInMs) {
 
         PerformanceMonitor.startActivity("Update Tick");
```

Comment:
delta is in seconds (e.g. 0.02 seconds for a frame at 50 frames a second).
delta * 1000 is in milliseconds (e.g. 0.02 * 1000 = 20 milliseconds).
So tick is in milliseconds. Not sure how it is used, but I guess it is a remainder from my original sweep of changing all deltas from milliseconds to seconds (easier to work with for game logic IMO).

You can use ```try (Closable c = new Closable()) {}``` blocks in Java. The problem is the PerformanceMonitor framework was created before we switched to Java 7 that introduced it, so hasn't been set up so that activities are a Closable object. I could add that though.

The longer winded alternative is:
```java
PerformanceMonitor.startActivity("Fun");
try {
  // ....
} finally {
  PerformanceMonitor.endActivity();
}```